cdk/kittyhawk/package.json

```diff
@@ -30,29 +30,29 @@
     "organization": "Penn Labs"
   },
   "devDependencies": {
-    "@types/jest": "^26.0.20",
-    "@types/node": "^10.17.0",
-    "@typescript-eslint/eslint-plugin": "^4.3.0",
-    "@typescript-eslint/parser": "^4.3.0",
-    "cdk8s-cli": "^1.0.0-beta.5",
-    "codecov": "^3.8.1",
-    "cron-time-generator": "^1.1.1",
-    "eslint": "^7.17.0",
-    "eslint-import-resolver-node": "^0.3.4",
-    "eslint-import-resolver-typescript": "^2.3.0",
-    "eslint-plugin-import": "^2.22.1",
-    "jest": "^26.6.3",
-    "jest-junit": "^12",
+    "@types/jest": "^27.0.2",
+    "@types/node": "^16.11.4",
+    "@typescript-eslint/eslint-plugin": "^5.1.0",
+    "@typescript-eslint/parser": "^5.1.0",
+    "cdk8s-cli": "^1.0.11",
```

Comment:
Hey! It turns out that for `cdk8s-cli`, after their `v1.0.0-beta.51` update they added in [input validation](https://github.com/cdk8s-team/cdk8s-cli/pull/67), which has a bug that's causing all the the `STRIPPED_BY_CDK8S` duplication in the code being generated. 

Bug github issue: https://github.com/cdk8s-team/cdk8s-cli/issues/108

There is a PR for `cdk8s-cli` that fixes it, but it's likely going to be a while before they merge, so until then I am going to bump this version down to the most recent version that doesn't have the input validation built in (`1.0.0-beta.50`).